upgrade-ert/pipeline.yml

```diff
@@ -62,7 +62,7 @@ jobs:
   - aggregate:
     - get: pivnet-product
       resource: elastic-runtime
-      trigger: true 
+      trigger: true
```

Comment:
It's good practice to only put things in the commit that the commit message declares. Even though we'd like it, it shouldn't be in this PR, so please remove it.